app/models/competition.rb

```diff
@@ -1564,7 +1564,7 @@ def ineligible_events(user)
   end
 
   def started?
-    start_date.present? && start_date < Date.today
+    start_date.present? && start_date <= Date.today
```

Comment:
Oh, this may have veeery unforessen consequences or side-effects. TBH I'm surprised that tests even pass. Why is this change necessary in the first place?

Reply:
> Why is this change necessary in the first place?

Currently, the check only considers a comp started if the current date is _after_ the start date - but, if the current date _is_ the start date, I think it is fair to consider a competition started. As it currently stands, a user could still edit their registration on the first day of a competition

> Oh, this may have veeery unforessen consequences or side-effects

Agreed - for a first step I'll take a look at where `.started?` is called and see if I notice any places this might cause issue

Reply:
Update: I only find 3 instances of `started?` being invoked - it might be worth confirming this using a grep on your side as well 
- `competition_spec.rb:202`
- `competition.rb:1173` (in the `registration_edits_currently_allowed?` method)
- In its method definition

Following `registration_edits_currently_permitted` leads to `can_edit_registrations?` in `user.rb`, which leads to a bunch of test cases and one use in `register.html.erb` - which I'm pretty sure is deprecated now that we're on the new registration system?

Reply:
Well then I suppose we should work on merging https://github.com/thewca/worldcubeassociation.org/pull/10411 first, because that will get rid of most `register.html.erb` code.

Reply:
Confirming that `can_edit_registrations?` is now only used in tests after merging in main - so the splash zone for changing `started?` is only the `registration_checker`, which is well-tested